connector/oidc/oidc.go

```diff
@@ -211,6 +229,15 @@ func (c *oidcConnector) LoginURL(s connector.Scopes, callbackURL, state string)
 	if s.OfflineAccess {
 		opts = append(opts, oauth2.AccessTypeOffline, oauth2.SetAuthURLParam("prompt", c.promptType))
 	}
+
+	if len(c.additionalAuthRequestParams) > 0 {
```

Comment:
This if is noop: if the list is empty, the loop won't go through it. Am I missing something?